lib/cws/server.ts

```diff
@@ -151,7 +153,7 @@ export class WebSocketServer extends eventEmitter() {
     return socket.end(`HTTP/1.1 ${code} ${name}\r\n\r\n`);
   }
 
-  private handleUpgrade(req: HTTP.IncomingMessage, socket: Socket): void {
+  private handleUpgrade(req: HTTP.IncomingMessage, socket: Socket, upgradeHead: any, callback: Function): void {
```

Comment:
Can you make `upgradeHead: any, callback: Function` optional `upgradeHead?: any, callback?: Function`

Reply:
If you set up `tslint` in your ide you should be able to see this issues :)